### ISSUE TITLE
Debt Collection Flag

### ISSUE THREAD

**Issue by tuckersamuelsen** [JustUtahCoders/utahexpungements.org#144]
Determine if the individual has had any case sent to the Office of State Debt Collection, or if they have a current balance on the case.

**Comment by benschinn** [comment]
@tuckersamuelsen if they have been sent to the Office of State Debt Collection, does that make the individual ineligible ?

**Comment by tuckersamuelsen** [comment]
> @tuckersamuelsen if they have been sent to the Office of State Debt Collection, does that make the individual ineligible ?

@benschinn not necessarily, but that is the closest we can come with the limitations of the court data.

For the purposes of this tool, we'll probably have flag the case if it has been sent at all.
